matchmake-extension/update_notification_data.go

```diff
@@ -42,7 +44,14 @@ func (protocol *Protocol) handleUpdateNotificationData(packet nex.PacketInterfac
 		return
 	}
 
-	err = uiParam1.ExtractFrom(parametersStream)
+	if libraryVersion.GreaterOrEqual("4.0.0") {
+		err = uiParam1.ExtractFrom(parametersStream)
+	} else {
+		var Param1 types.UInt32
```

Comment:
```suggestion
		var param1 types.UInt32
```

Can you make all variables like these lowercase?